app/email-incoming/circulars/index.ts

```diff
@@ -84,12 +85,16 @@ export const handler = createEmailIncomingMessageHandler(
       return
     }
 
+    const eventId = parseEventFromSubject(parsed.subject)
+    const synonyms = eventId ? [eventId] : []
     const circular = {
       subject: parsed.subject,
       body: parsed.text,
       sub: userData.sub,
       submitter: formatAuthor(userData),
       submittedHow,
+      eventId,
+      synonyms,
```

Comment:
Looks like you _are_ storing the synonyms in the Circulars table. Is this a mistake?

Reply:
you said not to update code before explaining the methodology. so I explained and did not update the code.

Reply:
Oh, I see. But you're planning on removing that?